curl.yaml

```diff
@@ -1,10 +1,16 @@
 package:
   name: curl
   version: 8.9.1
-  epoch: 1
+  epoch: 2
   description: "URL retrieval utility and library"
   copyright:
     - license: MIT
+  dependencies:
+    runtime:
+      - brotli-dev
+      - libpsl-dev
+      - nghttp2-dev
+      - openssl-dev
```

Comment:
these must be under curl-dev subpackage!!!!!!

Reply:
ah gotcha! I missed that part from your commit, thanks!